app/src/main-process/main.ts

```diff
@@ -46,10 +43,10 @@ if (__WIN32__ && process.argv.length > 1) {
   if (handleSquirrelEvent(process.argv[1])) {
     app.quit()
   } else {
-    const url = parseURL(process.argv[1])
-    if (url.name === 'open-repository') {
```

Comment:
@joshaber I wasn't sure about handling the OAuth callback here. We're tracking an issue around this in https://github.com/desktop/desktop/issues/1428 but I don't think "just send the info through" is the right behaviour.